Address feedback from internal test

MoLi7 opened the pull request.

WesIngwersen commented on CalculateEmissionFactors.Rmd:
````diff
@@ -92,3 +92,6 @@ table <- table[!substr(table[, paste(params$modeltype, "Code")], 1, 1)%in%c("S",
 write.csv(table, write_to_filename, row.names = FALSE)
 ```
+
+Saved `r write_to_filename`.
````
This at least gives the user the name of this file, although I see it the markdown converts the '_' to '\'

WesIngwersen reviewed (approved):
Works fine for me still - i added a couple more cosmetic changes

MoLi7 commented:
> Works fine for me still - i added a couple more cosmetic changes

Same here. I added a few more changes in README to further clarify usage options.

WesIngwersen commented on README.Rmd:
```diff
@@ -16,16 +16,18 @@ Code to produce the [Supply Chain GHG Emission Factors](https://cfpub.epa.gov/si
-the _CalculateEmissionFactors.Rmd_ in the editor, click on *Knit* and select *Knit with Parameters…*.
+If users choose to download a zip file of the code, they can unzip the file then [create a project from the existing directory within RStudio](https://support.rstudio.com/hc/en-us/articles/200526207-Using-RStudio-Projects) by selecting the unzipped local directory of files.
+
+If users choose to clone this repository, they can [create a project by cloing the repository with RStudio](https://resources.github.com/whitepapers/github-and-rstudio/#:~:text=Clone%20the%20repository%20with%20RStudio).
```
I think its should rather be

"If users have git installed, the users can create a project by cloning the repository with RStudio"

MoLi7 replied:
```suggestion
If users have git installed, they can [create a project by cloning the repository with RStudio](https://resources.github.com/whitepapers/github-and-rstudio/#:~:text=Clone%20the%20repository%20with%20RStudio).
```

WesIngwersen commented on README.Rmd:
```diff
@@ -16,16 +16,18 @@ Code to produce the [Supply Chain GHG Emission Factors](https://cfpub.epa.gov/si
+
+If users choose to clone this repository, they can [create a project by cloing the repository with RStudio](https://resources.github.com/whitepapers/github-and-rstudio/#:~:text=Clone%20the%20repository%20with%20RStudio).
+
+Once a project is created, double click to open it, then open the *CalculateEmissionFactors.Rmd* in the editor, click on *Knit* and select *Knit with Parameters…*.
```
I don't have to open the project by double-clicking ... once its created its automatically opened

MoLi7 replied:
```suggestion
Once a project is created and opened, then open the *CalculateEmissionFactors.Rmd* in the editor, click on *Knit* and select *Knit with Parameters…*.
```

MoLi7 replied:
Under the 'clone' option, users have to double click to open it.
